### PR TITLE
MYB 437 

### PR DESCRIPTION
**Pull request by julietapoume**
Add alert to ask if the user want to lose the actual results in the map with results of search, when the user make a long press

myb-437

### CODE REVIEW

**Comment by ignaciodevspark** [comment]
Looking sharp! Please merge..

File: MyBus/MyBus/MainViewController.swift [hunk 1]
```diff
@@ -207,22 +207,40 @@ class MainViewController: UIViewController {
...
-                        self.newDestination(destination)
-                    } else {
-                        self.newOrigin(destination)
-                    }
-                    self.homeNavigationBar(self.mapViewModel)
+            
+            if let annotations = self.mapViewController.mapView.annotations {
+                if( annotations.count > 1 ){
+                    self.progressNotification.stopLoadingNotification(self.view)
+                    let alert = UIAlertController(title: "Borrando busqueda actual", message: "Está seguro que quiere borrar la busqueda actual?", preferredStyle: UIAlertControllerStyle.ActionSheet)
```
Comment (marcosjvivar) [inline MyBus/MyBus/MainViewController.swift]: Can we add this string into the current implementation from @ignaciodevspark and update this lines of code?

**Comment by safink** [comment]
Good work. Approved 👍

File: MyBus/MyBus/MainViewController.swift (hunk 1 above)
Comment (lisandro) [inline MyBus/MyBus/MainViewController.swift]: Messages text must be in Localizable strings file. Please don't close current PR to do changes, just commit & push 👍

**Review by lisandro — CHANGES_REQUESTED** [review]

**Review by marcosjvivar — CHANGES_REQUESTED** [review]

**Pull request closed without merging**